consensus/blake3/consensus.go

```diff
@@ -861,10 +857,10 @@ func verifyInsideLocation(location []byte, number []*big.Int, config *params.Cha
 
 // Verifies that Location is valid inside current MapContext ontology.
 func checkInsideCurrent(regionLoc int, zoneLoc int, ontology []int) error {
-	if len(ontology) < regionLoc {
+	if regionLoc < 0 || regionLoc > ontology[0] {
```

Comment:
Should ontology be a global constant? Its not like the node would want to change ontologies between builds right?

Reply:
i.e. would a node sometimes call this function with `ontology := [2, 2]` and maybe another time call it with `ontology := [3, 3]` or something else? If not, I'd argue `ontology` should be a constant, not an arg 🤷

Reply:
Ontology should be a variable because eventually we will add code that allows ontology to change in a node based on historic tx throughput.  ie new allowed blockchains.